src/deprecated.jl

```diff
@@ -22,3 +22,6 @@ import Base: keys, values, insert!
 
 @deprecate stackdf stackdt
 @deprecate meltdf meltdt
+
+@deprecate readtable CSV.read
```

Comment:
can match in `CSV.write`
`separator -> delim`
`quotemark -> quotechar`
`header -> header`
`nastring -> null`

~no match in `CSV.write`~

no match in `writetable`
`?? -> escapechar`
`?? -> dateformat`
`?? -> colnames`
`?? -> append`
`?? -> transforms`